about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>About</title>
+</head>
+<body>
+
+  <div class="mobile-logo-wrap">
```

Comment:
- [ ] Your about page is missing this section 👇🏽 . Kindly add it

![image](https://user-images.githubusercontent.com/63473584/197053894-deb66b4a-7f98-42a5-95a6-571e72d8d600.png)